lxd/storage/drivers/driver_zfs_volumes.go

```diff
@@ -2271,8 +2271,10 @@ func (d *zfs) readonlySnapshot(vol Volume) (string, revert.Hook, error) {
 		return "", nil, err
 	}
 
+	dataset := d.dataset(vol, false)
+	srcSnapshot := fmt.Sprintf("%s@backup-%s", dataset, uuid.New())
```

Comment:
Can we use something other than "backup" here, it seems like this is left over from when it was only used as a temporary snapshot for backup purposes, but this isn't the case any more as its used for migrations also?

This only a temporary volume right? So changing the name isn't going to affect anything?

Maybe we could use "temp" instead?